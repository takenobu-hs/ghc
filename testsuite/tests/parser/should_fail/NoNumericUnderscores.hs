{-# Language NoNumericUnderscores #-}

module NoNumericUnderscores where

f :: Int -> ()
f 1_000 = ()
f _   = ()

