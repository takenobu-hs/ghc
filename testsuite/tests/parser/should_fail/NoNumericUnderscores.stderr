
NoNumericUnderscores.hs:6:1: error:
    Equations for ‘f’ have different numbers of arguments
      NoNumericUnderscores.hs:6:1-12
      NoNumericUnderscores.hs:7:1-10
